core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -909,7 +935,7 @@ class HttpGoogleServicesDAO(
     s"${Option(message).getOrElse("")} - code ${code}"
   }
 
-  private def addPolicyBindings(projectName: RawlsBillingProjectName, policiesToAdd: Map[String, List[String]]): Future[Unit] = {
+  private def removePolicyBindings(projectName: RawlsBillingProjectName, policiesToRemove: Map[String, Seq[String]]): Future[Unit] = {
```

Comment:
did you reorder the addPolicyBindings and removePolicyBindings functions in the file? This diff is confusing...

Reply:
I had to add `addPolicyBindings` to the `GoogleServicesDAO` trait and moved down its implementation in `HttpGoogleServicesDAO` so it'd be together with the other `override def`s. I didn't realize that caused a confusing diff. Sorry about that.